src/Modules/AuthenticationModule/components/Register/Fake.tsx

```diff
@@ -0,0 +1,258 @@
+import axios from "axios";
+import React, { useRef, useState } from "react";
+import { useForm, SubmitHandler } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import logo from "../../../../assets/images/PMS 3.png";
+import Styles from "./Register.module.css";
+
+import img from "../../../..//assets/images/8550fbcbe60cd242d12760784feff287.jpeg";
+
+interface IFormInput {
+    userName: string;
+    email: string;
+    country: string;
+    phoneNumber: string;
+    profileImage: FileList;
+    password: string;
+    confirmPassword: string;
+}
+
+export default function Register() {
+    const [visible, setVisible] = useState(false);
+
+    const navigate = useNavigate();
+    let {
+        register,
+        handleSubmit,
+        formState: { errors },
+        watch,
+    } = useForm<IFormInput>();
+    const imgValue = watch();
+
+    const password = useRef({});
+    password.current = watch("password", "");
+    const appendToFormData = (data: any) => {
+        const formData = new FormData();
+        formData.append("userName", data.userName);
+        formData.append("email", data.email);
+        formData.append("country", data.country);
+        formData.append("phoneNumber", data.phoneNumber);
+        formData.append("profileImage", data.profileImage[0]);
+        formData.append("password", data.password);
+        formData.append("confirmPassword", data.confirmPassword);
+        return formData;
+    };
+    const onSubmit: SubmitHandler<IFormInput> = async (data) => {
+        try {
+            const registerFormData = appendToFormData(data);
+
+            const response = await axios.post(
+                "https://upskilling-egypt.com:3003/api/v1/Users/Register",
+                registerFormData
+            );
+            console.log(response);
+
+            navigate("/verify");
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    return (
+        <>
+            <div className={` ${Styles.authcontainer}  `}>
```

Comment:
We are not using any MUI components in this page.

Reply:
updated whole register